app/components/forms/admin/settings/system-form.js

```diff
@@ -1,8 +1,186 @@
 import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
 
 const { Component } = Ember;
 
-export default Component.extend({
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        app_name: {
+          identifier : 'app_name',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter the App name')
+            }
+          ]
+        },
+        
+        tag_line: {
+          identifier : 'tag_line',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter a tag line')
+            }
+          ]
+        },
+
+        google_storage_bucket_name: {
+          identifier : 'google_storage_bucket_name',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter the Bucket name')
+            }
+          ]
+        },
+
+        google_storage_access_key: {
+          identifier : 'google_storage_access_key',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter the access key')
+            }
+          ]
+        },
+
+        google_storage_access_secret: {
+          identifier : 'google_storage_access_secret',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter the access secret')
+            }
+          ]
+        },
+
+        amazon_s3_region: {
+          identifier : 'amazon_s3_region',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please select a region')
+            }
+          ]
+        },
+
+        amazon_s3_bucket_name: {
+          identifier : 'amazon_s3_bucket_name',
+          rules      : [
+            {
+              type   : 'empty',
```

Comment:
For all this, also add `required=true` to the related form fields. for correct markup